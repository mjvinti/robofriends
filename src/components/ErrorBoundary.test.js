import React from 'react';
import { shallow } from 'enzyme';

import ErrorBoundary from './ErrorBoundary';

it('renders ErrorBoundary', () => {
    expect(shallow(<ErrorBoundary children={{}} />)).toMatchSnapshot();
});

it('handles componentDidCatch', () => {
    const wrapper = shallow(<ErrorBoundary children={{}} />);
    wrapper.instance().componentDidCatch();
    expect(wrapper.state().hasError).toBeTruthy();
});