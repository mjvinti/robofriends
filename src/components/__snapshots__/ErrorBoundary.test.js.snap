// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorBoundary 1`] = `<Component />`;
